### PR TITLE
Refactor TO RUN to compiler arch: Go side.

### PR DESCRIPTION
**Pull request by brightcoder01**
The Go side mainly contains the code_gen component.
Add a placeholder implementation in the runtime layer for local run.

### CODE REVIEW

File: python/runtime/local/submitter.py [hunk 1]
```diff
@@ -160,6 +160,16 @@ def submit_local_explain(datasource,
                  model=model)
 
 
+def submit_local_run(
+    datasource,
+    select,
+    image_name,
+    params,
+    into):
+    print("Execute local run. datasource:{}, select: {}, image_name: {}, params: {}, into: {}".format(
```
Comment (weiguoz) [inline python/runtime/local/submitter.py]: Add a space behind `datasource:` to unify the format.

File: go/codegen/experimental/codegen_step_run.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	err = runTemplate.Execute(&program, filter)
+	if err != nil {
+		return "", err
+	}
+
+	return program.String(), nil
+}
+
+const runStepTemplate = `
+def step_entry_{{.StepIndex}}():
```
Comment (sneaxiy) [inline go/codegen/experimental/codegen_step_run.go:62]: What if `TO RUN` is not running a Python program? For example, a C/C++ executable? Of course, we can call the C/C++ executable in `step_entry_0` on the Python side. But it requires that the docker image must install Python.

Comment (brightcoder01) [reply go/codegen/experimental/codegen_step_run.go:62]: Good question. Currently we have the requirement that the runnable docker image is built based on sqlflow/sqlflow:step. It has runtime library inside this image which already contains python.

File: python/runtime/local/submitter.py (hunk 1 above)
Comment (brightcoder01) [reply python/runtime/local/submitter.py]: Good catch, updated.

**Review by sneaxiy — APPROVED** [review]

**Pull request merged**